api/v2/views/machine_request.py

```diff
@@ -40,10 +42,10 @@ def get_queryset(self):
                     Q(start_date__gt=timezone.now() - timedelta(days=7))
                 )
             )
-            if self.request.user.is_staff:
+            if request_user.is_staff:
```

Comment:
Is `request_user` injects by DRF? I thought it was `request.user` that was the associated user object.